### PR TITLE
Tag Rotations v0.3.0 [https://github.com/FugroRoames/Rotations.jl]

### PR DESCRIPTION
**Pull request by andyferris**
New v0.5 - only version, based on *StaticArrays.jl*. Each rotation is now a 3x3 matrix (computed/sparse).

### CODE REVIEW

**Comment by andyferris** [comment]
FYI @awbsmith

**Comment by MichaelHatherly** [comment]
Since you're importing `Compat` it should probably be listed in the `REQUIRE` file as well.

**Comment by andyferris** [comment]
Good catch. I removed the `using Compat` line from my code and updated the SHA. Thanks.

File: Rotations/versions/0.3.0/sha1 [hunk 1]
```diff
@@ -1 +1 @@
-6196140ef7377b3f2f2cf18363d1413a49f056c8
+760f1fa3100adbfea9916630ead7cfa3abfeb503
```
Comment (tkelman) [inline Rotations/versions/0.3.0/sha1]: can you fix up the git tag too? https://github.com/FugroRoames/Rotations.jl/tree/v0.3.0

Comment (andyferris) [inline Rotations/versions/0.3.0/sha1]: Done!

**Comment by andyferris** [comment]
Should be good-to-go now

**Pull request merged**